profusion/sgqlc
# Added support to UUID

## Description


- Added support to `UUID` type. 
- Added documentation for it.

## Related Issues

Closes & Solves : #223 

## Progress


- [x] New type created
- [x] Documentation added and updated



### Pull request checklist




## How to test it


Now the example given in #223 works as expected. 

You can check via python:
Note: You can find more in doctests
```
>>> from sgqlc.types import Type
>>> from sgqlc.types.uuid import UUID

>>> class UUIDTestType(Type):
...     uuid=UUID
... 
>>> UUIDTestType
type UUIDTestType {
  uuid: UUID
}
>>> json_data = {'uuid': UUID('94fda4fb-d574-470b-82e2-0f4ec2a2db20')}
>>> obj = UUIDTestType(json_data)
>>> obj.uuid
UUID('94fda4fb-d574-470b-82e2-0f4ec2a2db20')

```

## Visual reference


Tested with the example given in https://github.com/profusion/sgqlc/issues/223 ( with system ) :
<img width="1061" alt="Screenshot 2023-02-27 at 19 21 00" src="https://user-images.githubusercontent.com/34503733/221649979-19ba6bdc-59ad-4a3c-806c-a73e3b4f3a5c.png">
